x-pack/plugins/actions/server/saved_objects/mappings.json

```diff
@@ -36,6 +36,9 @@
       "apiKey": {
         "type": "binary"
       },
+      "executionId": {
```

Comment:
interesting!  side effect of this PR is that we'll get the executionId into the ATP SO.  I'm not sure we ever have to "debug" these SO's, or haven't in a while, but that does now give us something to track them back to.